### DISCUSSION TITLE
How to obtain training data for stretching?

### DISCUSSION THREAD

**Discussion by ydmao4** [deepmodeling/deepmd-kit#2576] [Q&A]
I want to do a MD to stretch copper based graphene composite material. What kind of training data (from vasp) should be collected to obtain the potential?

**Comment by njzjz** [comment]
I don't know about your system, so I can only discuss some general ideas. Some short-time small-scale MD simulations (with any potential) similar to your target MD can be a start to be sampled from. These data should be labeled with high-accuracy first-principle methods. Then you can apply the DP-GEN scheme to collect more data.
